src/schema.rs

```diff
@@ -157,6 +164,10 @@ pub enum Type {
     String(SizeLength),
     ContractName(SizeLength),
     ReceiveName(SizeLength),
+    ULeb128(u32),
+    ILeb128(u32),
+    ByteList(SizeLength),
+    ByteArray(u32),
```

Comment:
It would be useful to start adding some documentation here. The names are getting very cryptic.

Reply:
I have added comments